packages/backend-deployer/src/cdk_deployer.ts

```diff
@@ -126,7 +127,7 @@ export class CDKDeployer implements BackendDeployer {
     }
     try {
       await this.executeChildProcess(
-        'npx',
+        this.packageManager === 'npm' ? 'npx' : this.packageManager,
```

Comment:
Can we do this in ctor and save result in private prop, so that we eliminate clones of this logic?

Reply:
This PR focus on adding the test. I'll make a follow-up PR to change the code